contrib/automation_tests/core/orbit_e2e.py

```diff
@@ -132,6 +132,13 @@ def _check_process_is_running(self, name: str):
         return False
 
     def _execute(self):
+
```

Comment:
We discussed waiting for the capture options button to become active here. 

That would assure that the wait_for_condition below does not accidentally pass before a "Finalizing Capture" window becomes displayed.

Reply:
We cannot really do that, because orbit_instance_connection.py only uses the Connection Window and never opens the OrbitMainWindow.

Reply:
Right, good catch.